functions/Invoke-DbaAdvancedUpdate.ps1

```diff
@@ -32,6 +32,9 @@ Function Invoke-DbaAdvancedUpdate {
     .PARAMETER ExtractPath
         Lets you specify a location to extract the update file to on the system requiring the update. e.g. C:\temp
 
+    .PARAMETER ArgumentList
+        A list of extra arguments to pass to the execution file.
+
```

Comment:
Your example includes the forwardslash, should we add that tidbit or template example to do parameter description?

It may be assumed but just want to check.

Reply:
That's a good idea. Even though there is now a new example, it would be beneficial to have that info in both places.

Reply:
I've updated the parameter in the user-facing command. Invoke-DbaAdvancedUpdate is never meant to be run by a user and is only public due to runspace-related requirements.

Reply:
Since it somehow failed on an unrelated test I re-pushed the commit that updates help for both commands.